alamode.js

```diff
@@ -2257,7 +2260,6 @@ var alamode = {
   },
 
   addLinkToBigNumber: function(o) {
```

Comment:
You could do 
`window["ALAMODE_CHARTS"] = window["ALAMODE_CHARTS"] || {};`
here if you weren't thrilled about the self-invoking function.

Reply:
Oh true! Thank you!